__tests__/app.test.js

```diff
@@ -50,6 +50,31 @@ describe("/api/topics", () => {
 });
 
 describe("/api/articles", () => {
+  describe("GET", () => {
+    test("200 + returns all articles", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          articles.forEach((article) => {
```

Comment:
As you're asserting within a forEach, could you please check the length of the array beforehand? 👍